Unit tests failing

The unit tests are currently failing on Python 3.x.  This can be seen in the travis builds:

```
=================================== FAILURES ===================================

_______________ TestMetaExtract.test__extract_to_tempdir_no_file _______________

self = <metaextract.tests.test_metaextract.TestMetaExtract object at 0x7f52c27e7710>

    def test__extract_to_tempdir_no_file(self):

        with pytest.raises(Exception) as e_info:

            with meta_utils._extract_to_tempdir("foobar"):

                pass

>       assert "foobar" in str(e_info)

E       AssertionError: assert 'foobar' in '<ExceptionInfo Exception tblen=3>'

E        +  where '<ExceptionInfo Exception tblen=3>' = str(<ExceptionInfo Exception tblen=3>)

metaextract/tests/test_metaextract.py:76: AssertionError

_______________________ TestMetaExtract.test_no_setup_py _______________________

self = <metaextract.tests.test_metaextract.TestMetaExtract object at 0x7f52c27a2a20>

tmpdir = local('/tmp/pytest-of-travis/pytest-0/test_no_setup_py0')

    def test_no_setup_py(self, tmpdir):

        with pytest.raises(Exception) as e_info:

            meta_utils._setup_py_run_from_dir(tmpdir.strpath, sys.executable)

>       assert tmpdir.strpath in str(e_info)

E       AssertionError: assert '/tmp/pytest-of-travis/pytest-0/test_no_setup_py0' in '<ExceptionInfo Exception tblen=2>'

E        +  where '/tmp/pytest-of-travis/pytest-0/test_no_setup_py0' = local('/tmp/pytest-of-travis/pytest-0/test_no_setup_py0').strpath

E        +  and   '<ExceptionInfo Exception tblen=2>' = str(<ExceptionInfo Exception tblen=2>)

metaextract/tests/test_metaextract.py:142: AssertionError

===================== 2 failed, 14 passed in 2.09 seconds ======================
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,12 +2,12 @@ sudo: false
 language: python
 matrix:
   include:
-  - python: 3.6
+  - python: 3.7
     env: TOX_ENV=pep8
   - python: 2.7
     env: TOX_ENV=py27
-  - python: 3.6
-    env: TOX_ENV=py36
+  - python: 3.7
+    env: TOX_ENV=py37
 install:
 - pip install tox
 script:

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,7 +2,7 @@ sudo: false
 language: python
 matrix:
   include:
-  - python: 2.7
+  - python: 3.6
     env: TOX_ENV=pep8
   - python: 2.7
     env: TOX_ENV=py27

--- a/metaextract/tests/test_metaextract.py
+++ b/metaextract/tests/test_metaextract.py
@@ -73,7 +73,7 @@ class TestMetaExtract(object):
         with pytest.raises(Exception) as e_info:
             with meta_utils._extract_to_tempdir("foobar"):
                 pass
-        assert "foobar" in str(e_info)
+        assert "foobar" in str(e_info.value)
 
     def test__extract_to_tempdir_tar_archive(self, tararchive):
         tarball_name, tarball_files = tararchive
@@ -139,7 +139,7 @@ class TestMetaExtract(object):
     def test_no_setup_py(self, tmpdir):
         with pytest.raises(Exception) as e_info:
             meta_utils._setup_py_run_from_dir(tmpdir.strpath, sys.executable)
-        assert tmpdir.strpath in str(e_info)
+        assert tmpdir.strpath in str(e_info.value)
 
     @pytest.mark.parametrize("fixture_name,expected_data", [
         (

--- a/tox.ini
+++ b/tox.ini
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py36,pep8
+envlist = py27,py37,pep8
 minversion = 1.6
 skipsdist = True
 

